### DISCUSSION TITLE
How to detect light or dark mode?

### DISCUSSION THREAD

**Discussion by CalaW** [alshedivat/al-folio#2476] [Q&A] [answered]
Hi, I have a logo with transparent background on my website.
There are two variants of the logo, one uses black text and the other uses white text.
I want to change the logo according to the theme of the website - use black text variant when in light mode, and vice versa.
Can someone tell me how to detect light or dark mode and change the logo accordingly?

**Comment by george-gca** [comment, marked as answer]
This is mostly handled in [assets/js/theme.js](https://github.com/alshedivat/al-folio/blob/master/assets/js/theme.js).

**Reply by CalaW** [reply]
Thanks! The final implementation was add an id for each logo, and toggle its display style in `_themes.scss`.

**Comment by sahiljhawar** [comment]
How to toggle between only light and dark mode?

I dont want this: 
<img width="52" alt="image" src="https://github.com/alshedivat/al-folio/assets/55475299/247e9d96-8e4b-4c5c-aa10-0ecdeaa06338">

**Reply by george-gca** [reply]
I meant you should open in its own Q&A. Q&As are meant to be more focused answers per specific case, and this way someone who is looking for the same solution to your question might not find it because it is hidden inside a related (but not exactly same) topic.

To remove the system theme setting, you should do the following changes:

![image](https://github.com/alshedivat/al-folio/assets/31376482/758ef01e-5923-47db-bbc3-ddfa8cfd30c8)

**Reply by sahiljhawar** [reply]
Alright, thanks.
